### PR TITLE
Enabling the use of BGRA in addition to BGR

### PR DESCRIPTION
**Pull request by DanielArnett**
As well as the option of adding a second texture to render.

### CODE REVIEW

**Comment by DanielArnett** [comment]
[Check this line before merging.](https://github.com/mohammedari/opengl_ros/blob/238f056096a46455b6ad575669a7d735166ff880/opengl_ros_lib/src/simple_renderer.cpp#L130)  It doesn't seem to be causing problems, but I wanted to make sure it was okay with you that I'm setting it to use GL_SRGB8_ALPHA8 instead of GL_SRGB8 in OpenGL.

File: opengl_ros_lib/src/simple_renderer.cpp [hunk 1]
```diff
@@ -175,7 +183,18 @@ void SimpleRenderer::Impl::render(cv::Mat& dest, const cv::Mat& src)
...
 
     //Read result
-    textureOut_.read(GL_BGR, GL_UNSIGNED_BYTE, dest.data, dest.rows * dest.cols * dest.channels());
+    textureOut_.read(glFormat_, GL_UNSIGNED_BYTE, dest.data, dest.rows * dest.cols * dest.channels());
+}
+
+void SimpleRenderer::Impl::render(cv::Mat& dest, const cv::Mat& src, const cv::Mat& secondSrc)
+{
+    cv::Mat localSrc = src.clone();
+    cv::Mat localSecond = secondSrc.clone();
```
Comment (mohammedari) [inline opengl_ros_lib/src/simple_renderer.cpp]: These two lines cause the error below in my environment.
```
/home/mohammedari/catkin_opengl2/devel/.private/opengl_ros_lib/lib/libopengl_ros_lib.so: undefined reference to `cv::_OutputArray::_OutputArray(cv::Mat&)'
```

**Comment by mohammedari** [comment]
Textures with the internal format of 'GL_SRGB8_ALPHA8' work with RGB input/output in my environment as I tested. It requires additional computational cost since glTexSubImage2D function converts RGB to RGBA, but it is trivial.

I think it would be OK to use GL_SRGB8_ALPHA8 for internal format. 
I would like to merge if build issue with clone() lines in simple_renderer.cpp is solved.

Many thanks!

**Comment by DanielArnett** [comment]
Okay I deleted those lines. They were only used for debugging before. Good catch!

**Pull request merged**